tomcatWarDeployer.py

```diff
@@ -243,55 +260,65 @@ def generateWAR(code, title, appname):
 </web-app>
 ''' % (title, appname.capitalize(), appname.capitalize(), appname))
 
-	cwd = os.getcwd()
-	os.chdir(dirpath)
-	outpath = tempfile.gettempdir() + '/' + appname + '.war'
-	logger.debug('Generating WAR file at: "%s"' % outpath)
-	packing = commands.getstatusoutput('jar -cvf %s *' % outpath)
-	os.chdir(cwd)
+    cwd = os.getcwd()
+    os.chdir(dirpath)
+    outpath = tempfile.gettempdir() + '/' + appname + '.war'
+    logger.debug('Generating WAR file at: "%s"' % outpath)
+
+    packing = None
+    if os.path.exists('/bin/jar'):
```

Comment:
This may not always be the case to have java archiver available in the /bin/ directory. One of the more popular approaches is to have many instances of JRE at various versions stored for instance on /opt/ - as it is in my machine. :-)
Therefore I would recommend to have the exists check tweaked a bit. 
Will look at it in some spare time.